src/components/member-task-update/index.js

```diff
@@ -34,10 +37,28 @@ const MemberTaskUpdate = () => {
     }
   };
 
+  const collapseTask = async (taskid) => {
+    setCollapsedState(true);
+    const data = { isCollapsed: !isCollapsed };
+    const res = await moveTask(taskid, data);
+    setCollapsedState(false);
+    if (res.status === 204) {
+      setTaskType(
+        `Task ${
+          isCollapsed ? TASK_LOOKS.EXPANDED : TASK_LOOKS.COLLAPSED
+        }! reloading...`
+      );
+      window.location.reload();
```

Comment:
why reload? is it not possible to do local state update?

Reply:
NO because i don't have Redux Store for state management

Reply:
```suggestion
      replace(asPath);
```
instead of window.location.reload()
ref: https://www.joshwcomeau.com/nextjs/refreshing-server-side-props/

Reply:
Done